<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::MainWindow</class>
 <widget class="QMainWindow" name="alt::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ALT Systeminfo</string>
  </property>
  <property name="windowIcon">
   <iconset theme="alterator"/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QDialogButtonBox" name="dialogButtonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="softwareTab">
       <attribute name="icon">
        <iconset theme="altlinux"/>
       </attribute>
       <attribute name="title">
        <string>About System</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="alt::OperationSystemPropertiesWidget" name="osPropertiesWidget" native="true"/>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>501</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hardwareTab">
       <attribute name="icon">
        <iconset theme="computer"/>
       </attribute>
       <attribute name="title">
        <string>Hardware</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="alt::HardwarePropertiesWidget" name="hardwarePropertiesWidget" native="true"/>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>501</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usefulTab">
       <attribute name="icon">
        <iconset theme="help"/>
       </attribute>
       <attribute name="title">
        <string>Release Notes</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="alt::UsefulSourcesWidget" name="usefulSourcesWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="icon">
        <iconset theme="gtk-preferences"/>
       </attribute>
       <attribute name="title">
        <string>Local Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="alt::SettingsWidget" name="settingsWidget" native="true"/>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="hostNameLabel">
        <property name="text">
         <string notr="true">Placeholder</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="Line" name="rightLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="Line" name="leftLine">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>alt::OperationSystemPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>ui/operationsystempropertieswidget/operationsystempropertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::HardwarePropertiesWidget</class>
   <extends>QWidget</extends>
   <header>ui/hardwarepropertieswidget/hardwarepropertieswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::UsefulSourcesWidget</class>
   <extends>QWidget</extends>
   <header>ui/usefulsourceswidget/usefulsourceswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>alt::SettingsWidget</class>
   <extends>QWidget</extends>
   <header>ui/settingswidget/settingswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../logo.qrc"/>
  <include location="../../logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>alt::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
